Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -252,4 +245,20 @@ - (SentrySpanStatus)spanStatusForHttpResponseStatusCode:(NSInteger)statusCode
     return kSentrySpanStatusUndefined;
 }
 
+- (nullable NSDictionary *)addTraceHeader:(nullable NSDictionary *)headers
+{
+    id<SentrySpan> span = SentrySDK.currentHub.scope.span;
+    if (span == nil) {
+        return headers;
+    }
+
```

Comment:
Perhaps assign the resulting dictionary to a variable and after the `if` you could do:

If we have a check for the log level, we can do this:
```
if (SentrySDK.logger.isEnabled(SentryLevel.Debug)) {
// Expensive operations done just for logging go here
}
```

Reply:
Will add that in the future.